docs/source/python/install.rst

```diff
@@ -93,3 +100,41 @@ a custom path to the database from Python:
 
    >>> import pyarrow as pa
    >>> pa.set_timezone_db_path("custom_path")
+
+
+.. _python-conda-differences:
+
+Differences between conda-forge packages
+----------------------------------------
+
+PyArrow is packaged on `conda-forge <https://conda-forge.org/>`_ as three
+separate packages, each providing varying levels of functionality. This is in
+contrast to PyPi, where only a single PyArrow package is provided.
+
+The purpose of this split is to minimize the size of the installed package for
+most users (``pyarrow``), provide a smaller, minimal package for specialized use
+cases (``pyarrow-core``), while still providing a complete package for users who
+require it (``pyarrow-all``).
+
+The table below lists the functionality provided by each package and may be
+useful when deciding to use one package over another:
+
++------------+------------------------------+------------------------------+------------------------------+
+| Component  | pyarrow                      | pyarrow-core                 | pyarrow-all                  |
++============+==============================+==============================+==============================+
+| Core       | :fas:`check;sd-text-success` | :fas:`check;sd-text-success` | :fas:`check;sd-text-success` |
++------------+------------------------------+------------------------------+------------------------------+
+| Parquet    | :fas:`check;sd-text-success` |                              | :fas:`check;sd-text-success` |
++------------+------------------------------+------------------------------+------------------------------+
+| Datasets   | :fas:`check;sd-text-success` |                              | :fas:`check;sd-text-success` |
++------------+------------------------------+------------------------------+------------------------------+
+| Acero      | :fas:`check;sd-text-success` |                              | :fas:`check;sd-text-success` |
++------------+------------------------------+------------------------------+------------------------------+
+| Substrait  | :fas:`check;sd-text-success` |                              | :fas:`check;sd-text-success` |
++------------+------------------------------+------------------------------+------------------------------+
+| Flight     |                              |                              | :fas:`check;sd-text-success` |
++------------+------------------------------+------------------------------+------------------------------+
+| Flight SQL |                              |                              | :fas:`check;sd-text-success` |
++------------+------------------------------+------------------------------+------------------------------+
+| Gandiva    |                              |                              | :fas:`check;sd-text-success` |
```

Comment:
Note to reviewers: I opted for a font-awesome green checkmark here instead of an emoji since emoji will render differently across platforms whereas font-awesome will look the same. Feel free to bike shed this.

Reply:
Does the checkmark from the implementation status page fall under emojis?
https://raw.githubusercontent.com/apache/arrow/main/docs/source/status.rst

I like using font-awesome otherwise. Can't wait to see the preview build and check how it looks like =)

Reply:
In the status.rst we use unicode checkmark character, so that's also an option:

https://github.com/apache/arrow/blob/b3b4521852384e842e9bcb6d4e05d7c11669674f/docs/source/status.rst?plain=1#L31-L37

Reply:
Ah, yes. I think the unicode check is going to be a lot more consistent/predictable.

With the above code, the table looks like this:

![Screenshot 2024-04-11 at 11 21 53 AM](https://github.com/apache/arrow/assets/563/2de83fe5-af7c-4f55-8022-cd6245dcae53)

We should probably use the same checkmark as the implementation status just for consistency. I'll change this to use  ✓ unless either of you vote strongly in favor of the font-awesome checkmark :)

Reply:
Green font-awesome checkmarks are really nice but I would still vote for consistency and the use of unicode checkmark character 😊

Reply:
Reverted these to ✓ in https://github.com/apache/arrow/pull/41135/commits/cccc7ff655ae34420214d3b512aafaebc6775942.